db/migrate/20210830070917_add_upstream_for_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddUpstreamForRules < ActiveRecord::Migration[5.2]
+  def change
+    add_column :rules, :upstream, :boolean, null: false, default: true
+  end
```

Comment:
An index may be warranted:

```suggestion
    add_column :rules, :upstream, :boolean, null: false, default: true
    add_index :rules, :upstream
  end
```